core/templates/pages/release-coordinator-page/jobs-tab/jobs-tab.component.ts

```diff
@@ -62,22 +63,23 @@ export class AdminJobsTabComponent {
   startNewJob(
       jobType: string): void {
     this.setStatusMessage.emit('Starting new job...');
-    this.adminBackendApiService.startNewJobAsync(jobType).then(() => {
-      this.setStatusMessage.emit('Job started successfully.');
-      this.windowRef._window().location.reload();
-    }, (errorResponse) => {
-      this.setStatusMessage.emit(
-        'Server error: ' + errorResponse.error
-      );
-    });
+    this.releaseCoordinatorBackendApiService.startNewJobAsync(jobType)
+      .then(() => {
+        this.setStatusMessage.emit('Job started successfully.');
```

Comment:
Can we please use snackbars instead? They're very pretty :)

https://material.angular.io/components/snack-bar/overview

Reply:
I'm trying to reduce the size of this PR and have seperate PR to add new design for jobs tab (Also, the existing bar makes it look consistent to admin page.)

Reply:
I tried adding it and looks looks I've too add a angualr structure to bind this with other directives/component as we set this message from other directives as well in the page. Will it be fine to work on this in the PR where we are redesgining the job's component?